### DISCUSSION TITLE
如何查询更新日志

### DISCUSSION THREAD

**Discussion by asforest** [walinejs/waline#1561] [Q&A] [answered]
waline仓库没有Release页面，查找不了大版本的更新记录。文档里也找不到，每次发布新版本我都不敢更新，新版更新了什么内容我完全不知，生怕出兼容性问题。

请问除了写脚本过滤提交记录以外，还有方法能查询大版本的更新记录吗？比如1.20更新到1.21这样的记录。

**Comment by asforest** [comment]
对了，更新日志主要指服务端更新日志

**Comment by lizheming** [comment, marked as answer]
https://t.me/walinelog 或者看 QQ 用户群公告。一般 SQL 类存储服务需要关注下字段新增，默认存储服务都可以无脑更新。

**Reply by asforest** [reply]
明白了，谢谢！
